test/integration/change-streams/change_stream.test.js

```diff
@@ -187,6 +187,77 @@ describe('Change Streams', function () {
   });
   afterEach(async () => await mock.cleanup());
 
+  context('ChangeStreamCursor options', function () {
+    let client, db, collection;
+
+    beforeEach(async function () {
+      client = await this.configuration.newClient().connect();
+      db = client.db('db');
+      collection = db.collection('collection');
+    });
+
+    afterEach(async function () {
+      await client.close();
+      client = undefined;
+      db = undefined;
+      collection = undefined;
+    });
+
+    context('fullDocument', () => {
+      it('sets fullDocument to `undefined` if no value is passed', function () {
+        const changeStream = client.watch();
+
+        expect(changeStream).not.to.have.nested.property(
+          'cursor.pipeline[0].$changeStream.fullDocument'
+        );
+      });
+
+      it('assigns `fullDocument` to the correct value if it is passed as an option', function () {
```

Comment:
Should there be another test case that uses a random string and/or "default" so that if we add code to start validating the options we'll get it flagged as a breaking change?